src/python/pyImpactX.H

```diff
@@ -10,9 +10,10 @@
 #define IMPACTX_PYIMPACTX_H
 
 #include <pybind11/pybind11.h>
+#include <pybind11/functional.h>
+#include <pybind11/numpy.h>
```

Comment:
I am not even sure we still use this. Could be tried to be commented out once CI passes:
```suggestion
// include <pybind11/numpy.h>  // not yet used
```